packages/tools/devtools/devtools-core/src/ContainerDevtools.ts

```diff
@@ -534,11 +534,11 @@ export class ContainerDevtools implements IContainerDevtools, HasContainerKey {
 	 */
 	private getSupportedFeatures(): ContainerDevtoolsFeatureFlags {
 		return {
-			[ContainerDevtoolsFeature.ContainerData]: this.containerData !== undefined,
+			containerDataVisualization: this.containerData !== undefined,
```

Comment:
I approved but one last thing here... do we need to keep putting this value in the "container-data" key for the scenario of someone updating Devtools but still having an older version of the extension?

Reply:
This isn't strictly for backwards compatibility - it's also for the scenario where the app does not provide container data for us to visualize.

Reply:
```suggestion
			// If no container data was provided to the devtools, we cannot support data visualization.
			containerDataVisualization: this.containerData !== undefined,
```

Reply:
@alexvy86 Actually, maybe I'm not understanding the question. Could you elaborate a bit?

Reply:
So, if someone is using the currently published version of the extension, their extension expects to see a value under `"container-data"` in order to enable data visualization, right? And if they upgrade the devtools/devtools-core version they're using to one with this change, their extension will never again enable data visualization because the supported features message won't have the `"container-data"` key anymore, only the new key.

Reply:
Ah, okay, I understand. Yes, you're right. Good catch. I even had that initially and removed it... I don't remember why I thought it was okay to do so; it definitely isn't.